packages/calendar/test/lion-calendar.test.js

```diff
@@ -373,31 +287,6 @@ describe('<lion-calendar>', () => {
           // dates in the month view will be derived from.
           expect(isNormalizedDate(el.centralDate)).to.be.true;
         });
-
-        it('normalizes dates in date comparisons', async () => {
```

Comment:
This functionality should really still work and be tested.
Any reason for removing it?